projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -1050,6 +1074,7 @@ public function register_post_meta() {
 			register_meta( 'post', $this->POST_MESS, $message_args );
 			register_meta( 'post', $this->POST_TWEETSTORM, $tweetstorm_args );
 			register_meta( 'post', self::POST_PUBLICIZE_FEATURE_ENABLED, $publicize_feature_enable_args );
+			register_meta( 'post', self::POST_ATTACHED_MEDIA, $publicize_attached_media_args );
```

Comment:
I've just realised that we should be registering these meta values for any post type that supports Publicize. Maybe not tweetstorm as that's a bit specific to posts, but the other values would be needed.

Reply:
`post` should cover all custom post types as well—you can specify `object_subtype` to add it just for a specific post type: https://developer.wordpress.org/reference/functions/register_meta/

Reply:
We're actually looping through all supported post types explicitly already: https://github.com/Automattic/jetpack/blob/trunk/projects/packages/publicize/src/class-publicize-base.php#L1041